examples/local/fivenodenetwork/main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/ava-labs/avalanche-network-runner/local"
 	"github.com/ava-labs/avalanche-network-runner/network"
 	"github.com/ava-labs/avalanchego/utils/logging"
+	"go.uber.org/zap"
```

Comment:
Note: in this PR https://github.com/ava-labs/avalanche-network-runner/pull/230 we are unifying ANR logging to use just the avalanchego logger (which also used zap, but behind), so depending on what gets merged first, someone might need to address.

Reply:
probably we may accept this, 230, and later make a new PR for introducing zap types